src/main/java/io/github/org/programming/bot/commands/thread/AskCommand.java

```diff
@@ -0,0 +1,102 @@
+            event.reply("You can only create 2 threads per day").setEphemeral(true).queue();
+            return;
+        }
+
+        // event.getTextChannel().createThreadChannel("Ask a question").queue();
+
+        var threadName = event.getOption("name", OptionMapping::getAsString);
+        var threadType = event.getOption("type", OptionMapping::getAsString);
+
+        if (threadName == null || threadType == null) {
+            event.reply("You must provide a name and a type for the thread")
+                .setEphemeral(true)
+                .queue();
+            return;
+        }
+
+        var thread = event.getGuild().getTextChannelById("1007915436156915732");
+
+        if (thread == null) {
+            event.reply("Could not find thread channel").setEphemeral(true).queue();
+            return;
+        }
+
+        var threadChannel = thread.createThreadChannel(threadName).complete();
+
+        threadChannel
+            .sendMessage(
+                    "Note: That if no one response, it might mean your question is to vague or "
+                            + "not clear enough.")
+            .setEmbeds(detail())
+            .queue();
+
+        updateAskDatabase(event.getMember().getId(), event.getGuild().getId());
+    }
+
+    private MessageEmbed detail() {
+        return new EmbedBuilder().setTitle("Please provide the following details")
+            .addField("Description", "Please provide a description of your problem", false)
+            .addField("Code",
+                    "Please provide the code where the problem is happening. Seem image below for example",
+                    false)
+            .setImage(
+                    "https://cdn.discordapp.com/attachments/1007751377306533898/1007751744308117524/codeSyntaxExample.png")
+            .setColor(Color.GRAY)
+            .build();
+    }
+
+    @Override
+    public SlashCommand build() {
+        return new SlashCommandBuilder("ask", "Ask a question")
+            .addOption(OptionType.STRING, "name", "The name of the question", true)
+            .addOptions(new OptionData(OptionType.STRING, "type", "The type of question", true)
+                .addChoices(choices))
+            .build()
+            .setToGuildOnly();
+    }
+
+    private List<Command.Choice> choices =
+            List.of(new Command.Choice("Java", "java"), new Command.Choice("C++", "c++"),
+                    new Command.Choice("C#", "c#"), new Command.Choice("Python", "python"),
+                    new Command.Choice("JavaScript", "js"), new Command.Choice("PHP", "php"),
+                    new Command.Choice("C", "c"), new Command.Choice("Go", "go"),
+                    new Command.Choice("Rust", "rust"), new Command.Choice("Swift", "swift"),
+                    new Command.Choice("Kotlin", "kotlin"), new Command.Choice("Scala", "scala"),
+                    new Command.Choice("TypeScript", "ts"), new Command.Choice("Other", "other"));
+}
```

Comment:
What is this ?

Reply:
when asking you do ```/ask name topic``` The topic will be used to differentiate in the active question channel

Reply:
topic will be a list of choices

Reply:
<img width="1666" alt="Screenshot 2022-08-13 at 11 28 27" src="https://user-images.githubusercontent.com/67903886/184479745-c9f53db1-f754-4f33-8ed9-fd4190f65509.png">
something like this

Reply:
might rename it to category makes more sense